lib/controller/home_controller.dart

```diff
@@ -1,122 +1,47 @@
-      accountList: <Account>[
-        Account(
-            title: 'Appartment Rental',
-            budget: 5000.0,
-            expense: 3000.0,
-            accountId: null,
-            userId: null,
-            summaryId: null),
-        Account(
-            title: 'Food',
-            budget: 4000.0,
-            expense: 3000.0,
-            accountId: null,
-            userId: null,
-            summaryId: null),
-        Account(
-            title: 'Motorcycle Bill',
-            budget: 5000.0,
-            expense: 3000.0,
-            accountId: null,
-            userId: null,
-            summaryId: null),
-        Account(
-            title: 'Others',
-            budget: 5000.0,
-            expense: 3000.0,
-            accountId: null,
-            userId: null,
-            summaryId: null),
-        Account(
-            title: 'Appartment Rental',
-            budget: 5000.0,
-            expense: 3000.0,
-            accountId: null,
-            userId: null,
-            summaryId: null),
-      ],
-    ),
-    //BUDGET 2
-    MonthlySummary(
-      month: 3,
-      year: 2021,
-      expense: 19000,
-      budget: 40000,
-      balance: 1000,
-      accountList: <Account>[
-        Account(
-            title: 'Appartment Rental',
-            budget: 5000.0,
-            expense: 3000.0,
-            accountId: null,
-            userId: null,
-            summaryId: null),
-        Account(
-            title: 'Food',
-            budget: 4000.0,
-            expense: 3000.0,
-            accountId: null,
-            userId: null,
-            summaryId: null),
-        Account(
-            title: 'Motorcycle Bill',
-            budget: 5000.0,
-            expense: 3000.0,
-            accountId: null,
-            userId: null,
-            summaryId: null),
-        Account(
-            title: 'Appartment Rental',
-            budget: 5000.0,
-            expense: 3000.0,
-            accountId: null,
-            userId: null,
-            summaryId: null),
-      ],
-    ),
+  final MonthlySummaryRepository _monthlySummaryRepository =
+      MonthlySummaryRepository();
 
-    //BUDGET 3
-    MonthlySummary(
-      month: 4,
-      year: 2021,
-      budget: 40000,
-      balance: 10000,
-      expense: 19000,
-      accountList: <Account>[
-        Account(
-            title: 'Motorcycle Bill',
-            budget: 5000.0,
-            expense: 3000.0,
-            accountId: null,
-            userId: null,
-            summaryId: null),
-        Account(
-            title: 'Others',
-            budget: 5000.0,
-            expense: 3000.0,
-            accountId: null,
-            userId: null,
-            summaryId: null),
-        Account(
-            title: 'Appartment Rental',
-            budget: 5000.0,
-            expense: 3000.0,
-            accountId: null,
-            userId: null,
-            summaryId: null),
-      ],
-    ),
-  ];
+  final RxBool _isLoading = false.obs;
```

Comment:
ang kining _isloading nga variable pwede rane dili nimo e rxbool kay sa imong pag implement sa UI kay  Get builder man imong gamit. Pwede ra sad nimo e extend ang basecontroller instead of getxcontroller kay naanay isloading adto nga ge implement.

Reply:
Copy brad..